### DISCUSSION TITLE
Is it possible to deploy  VolSync as a namespace operator rather than cluster based operator?

### DISCUSSION THREAD

**Discussion by scoutjohn** [backube/volsync#793] [Q&A] [answered]
Is it possible to deploy Volsync operator as a namespace operator? If I change ClusterRole and it biding to Role and Rolebinding, would it work? 
I have applications of the same kind from 2 different teams who are trying to replicate data.

The user we're using to deploy is not an admin user, does not have the privilege to deploy on a cluster level. Any help is appreciated.

**Comment by JohnStrunk** [comment, marked as answer]
VolSync isn't designed to be deployed at the Namespace level, though once it's installed, no permissions beyond Namespace-level are required to use it.

VolSync has associated ReplicationSource and ReplicationDestination CRDs that needs to be installed into the cluster. This is an admin-level operation, so I don't think there's any way around having elevated permissions during installation. The operator also watches all namespaces for RS and RD instances--- I believe a fairly simple code change could confine it to watching one NS, but that doesn't fix the CRD installation issue.
